proxy/stats_get_prediction.go

```diff
@@ -60,16 +60,10 @@ func (s *StatsGetPrediction) StatsGetStateHandler(next http.Handler) http.Handle
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		path := r.URL.Path
 		switch path {
-		case "/api/tus/read", "/api/sys/get_news":
-			if s.predictionState == sending_calls {
-				fmt.Println("Done looking up stats")
-			}
-			s.predictionState = ready
-			next.ServeHTTP(w, r)
 		case "/api/statistics/get":
-			if s.predictionState == ready {
-				body, _ := io.ReadAll(r.Body)
-				r.Body = io.NopCloser(bytes.NewBuffer(body))
+			body, _ := io.ReadAll(r.Body)
+			r.Body = io.NopCloser(bytes.NewBuffer(body))
+			if strings.HasSuffix(string(body), "07ffffffff\x00") {
```

Comment:
I assume `07ffffffff\x00` is the first R-Code call?

It might be better to pull it from the top of `ExpectedRCodeCalls()` so this always works as long as `ExpectedRCodeCalls` is kept up to date.

Reply:
Also removes duplicates of this magic string, since it's also used in `AsyncGetStats()`